src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -674,14 +701,15 @@ def __call__(
 
             layer_outputs = layer_module(
                 hidden_states,
-                attention_mask=attention_mask,
-                position_bias=position_bias,
-                encoder_hidden_states=encoder_hidden_states,
-                encoder_attention_mask=encoder_attention_mask,
-                encoder_decoder_position_bias=encoder_decoder_position_bias,
-                output_attentions=output_attentions,
-                deterministic=deterministic,
-                init_cache=init_cache,
+                attention_mask,
+                position_bias,
+                encoder_hidden_states,
+                encoder_attention_mask,
+                encoder_decoder_position_bias,
+                output_attentions,
+                return_dict,
```

Comment:
`return_dict` not in fact used by the `FlaxT5LayerCollection` - we can omit this arg (as was done previously)!